src/app/modules/color-scheme/color-scheme-contents/color-scheme-contents.component.html

```diff
@@ -0,0 +1,65 @@
+      <div class="square" (click)="colorChanged([2, 5])" [class.currentcolor]="colorSelectedStatus[2][5]"></div>
+      <div class="square" (click)="colorChanged([2, 6])" [class.currentcolor]="colorSelectedStatus[2][6]"></div>
+    </div>
+    <div class="swatch four" (click)="schemeChanged(3)">
+      <div class="square" (click)="colorChanged([3, 0])" [class.currentcolor]="colorSelectedStatus[3][0]"></div>
+      <div class="square" (click)="colorChanged([3, 1])" [class.currentcolor]="colorSelectedStatus[3][1]"></div>
+      <div class="square" (click)="colorChanged([3, 2])" [class.currentcolor]="colorSelectedStatus[3][2]"></div>
+      <div class="square" (click)="colorChanged([3, 3])" [class.currentcolor]="colorSelectedStatus[3][3]"></div>
+      <div class="square" (click)="colorChanged([3, 4])" [class.currentcolor]="colorSelectedStatus[3][4]"></div>
+      <div class="square" (click)="colorChanged([3, 5])" [class.currentcolor]="colorSelectedStatus[3][5]"></div>
+      <div class="square" (click)="colorChanged([3, 6])" [class.currentcolor]="colorSelectedStatus[3][6]"></div>
+    </div>
+    <div class="swatch five" (click)="schemeChanged(4)">
+      <div class="square" (click)="colorChanged([4, 0])" [class.currentcolor]="colorSelectedStatus[4][0]"></div>
+      <div class="square" (click)="colorChanged([4, 1])" [class.currentcolor]="colorSelectedStatus[4][1]"></div>
+      <div class="square" (click)="colorChanged([4, 2])" [class.currentcolor]="colorSelectedStatus[4][2]"></div>
+      <div class="square" (click)="colorChanged([4, 3])" [class.currentcolor]="colorSelectedStatus[4][3]"></div>
+      <div class="square" (click)="colorChanged([4, 4])" [class.currentcolor]="colorSelectedStatus[4][4]"></div>
+      <div class="square" (click)="colorChanged([4, 5])" [class.currentcolor]="colorSelectedStatus[4][5]"></div>
+      <div class="square" (click)="colorChanged([4, 6])" [class.currentcolor]="colorSelectedStatus[4][6]"></div>
+    </div>
+    <div class="swatch six" [class.selected]="schemeSelectedStatus[5]" (click)="schemeChanged(5)">
+      <div class="gradient" (click)="colorChanged([5, 0])"></div>
+    </div>
+    <div class="swatch seven" [class.selected]="schemeSelectedStatus[6]" (click)="schemeChanged(6)">
+      <div class="gradient" (click)="colorChanged([6, 0])"></div>
+    </div>
+    <div class="swatch eight" [class.selected]="schemeSelectedStatus[7]" (click)="schemeChanged(7)">
+      <div class="gradient" (click)="colorChanged([7, 0])"></div>
+    </div>
+    <div class="brightness-slider-label">Brightness</div>
+    <ng5-slider class="slider" [options]="options" [(value)]="brightness[0]" [(highValue)]="brightness[1]"
+      (userChangeEnd)="brightnessChanged()">
+    </ng5-slider>
+    <div class="transparency-slider-label">Transparency</div>
+    <ng5-slider class="slider" [options]="options" [(value)]="transparency" (userChangeEnd)="transparencyChanged()">
```

Comment:
This slider is missing the filled bar from the start to the current value. This is an option that needs tweaking.